lint-repository/action.yml

```
@@ -20,6 +20,12 @@ runs:
         fetch-depth: 0
         path: _repolint
 
-    - name: Run linter
+    - name: "check branch"
```

```suggestion
    - name: "Run linter"
```

I would call the job back because the point is to do it